src/main/features/claim/routes/pay.ts

```diff
@@ -94,12 +97,15 @@ async function successHandler (res, next) {
     }
 
     if (await featureTogglesClient.isAdmissionsAllowed(user, roles)) {
-      await claimStoreClient.saveClaim(draft, user, 'admissions')
+      savedClaim = await claimStoreClient.saveClaim(draft, user, 'admissions')
     } else {
-      await claimStoreClient.saveClaim(draft, user)
+      savedClaim = await claimStoreClient.saveClaim(draft, user)
     }
   }
+  const payClient: PayClient = await getPayClient(req)
+  const paymentReference = draft.document.claimant.payment.reference
 
+  payClient.update(user, paymentReference, savedClaim.externalId, savedClaim.claimNumber)
```

Comment:
i think you should have await start if this line...intellij should complain the way it is now

Reply:
done